### PR TITLE
✨ Support display grant tx memo

### PR DESCRIPTION
**Pull request by williamchong**
image
Requires data from https://github.com/likecoin/likecoin-api-public/pull/416

### CODE REVIEW

File: src/mixins/nft.js [hunk 1]
```diff
@@ -495,14 +495,42 @@ export default {
...
+                classId,
+                nftId,
+                fromWallet: granterWallet,
+                event: 'grant',
+                memo: granterMemo,
+                txHash: grantTxHash,
+                timestamp: timestamp + 1,
+              };
+              history.push(e);
+              history.sort((a, b) => b.timestamp - a.timestamp);
```
[the remark this answers is not in the document]
Comment (williamchong) [reply src/mixins/nft.js:527]: image
Array ordering affects message ordering. Not sure if should sort at the end of the loop or here

File: src/mixins/portfolio.js [hunk 2]
```diff
@@ -327,7 +327,7 @@ export const createPorfolioMixin = ({
     async loadNFTListByAddress(address) {
       const fetchPromise = Promise.all([
         this.fetchNFTListByAddress(address),
-        this.fetchNFTDisplayStateListByAddress(address),
+        // this.fetchNFTDisplayStateListByAddress(address),
```
[the remark this answers is not in the document]
Comment (williamchong) [reply src/mixins/portfolio.js:330]: Accident, reverted

File: src/mixins/nft.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nwingt) [reply src/mixins/nft.js:527]: should be sorted once at the end?

**Review by WeiJun0827 — APPROVED** [review]
Approved besides Tat's comment

File: src/mixins/nft.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamchong) [reply src/mixins/nft.js]: updated

File: src/mixins/portfolio.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nwingt) [reply src/mixins/portfolio.js:330]: Seems has not changed

[the remark this answers is not in the document]
Comment (williamchong) [reply src/mixins/portfolio.js]: True-reverted, also added a event filter to filter out grant message in event list

**Review by nwingt — APPROVED** [review]

**Review by AuroraHuang22 — APPROVED** [review]

**Pull request merged**